src/main/kotlin/config/WebSecurityConfig.kt

```diff
@@ -53,6 +53,7 @@ class WebSecurityConfig : WebSecurityConfigurerAdapter() {
             .antMatchers("/favicon.ico").permitAll()
             .antMatchers("/").permitAll()
             .antMatchers("/*.js").permitAll()
+            .antMatchers("/images/*.webp").permitAll()
```

Comment:
Why not images/*.*? In the future we might want to add other formats. (Like svg :)) 
Even though I agree that `.webp` should be used the most since it's probably the most efficient image format which also has good support at this time, I feel like explicitly disallowing all other formats is a bit restrictive. (Especially svg)

Reply:
I think it is good to be very specific when it comes to security